client/src/themes/theme.js

```diff
@@ -3,12 +3,12 @@ import { createMuiTheme } from "@material-ui/core";
 export const theme = createMuiTheme({
   typography: {
     fontFamily: '"Roboto"',
-    fontSize: 12,
+    fontSize: 16,
```

Comment:
I'll approve for now but isn't 16 kind of big? lol

Reply:
on second thought, it might actually be too big... I'll revert it to 12 😅